evm/src/trace/mod.rs

```diff
@@ -69,7 +73,7 @@ impl CallTraceArena {
                     None
                 }
             } else {
-                None
+                self.codes.get(&node.trace.address)
```

Comment:
We might as well only use this instead of the output of the creation node if we merge